jmclient/test/test_wallet_rpc.py

```diff
@@ -288,11 +288,18 @@ def process_direct_send_response(self, response, code):
     def process_wallet_display_response(self, response, code):
         assert code == 200
         json_body = json.loads(response.decode("utf-8"))
-        latest_balance = float(json_body["walletinfo"]["total_balance"])
+        wi = json_body["walletinfo"]
+        latest_balance = float(wi["total_balance"])
         jlog.info("Wallet display currently shows balance: {}".format(
             latest_balance))
         assert latest_balance > self.mean_amt * 4.0 - 1.1
         assert latest_balance <= self.mean_amt * 4.0 - 1.0
+        # these samplings are an attempt to ensure object structure:
+        wia = wi["accounts"]
+        assert wia[0]["branches"][0]["entries"][0]["label"] == ""
```

Comment:
Maybe it would be better to add some actual label and then test against non-empty value?

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1141/commits/393971446e2ded67989da6d21dedfee2578ca9d6

There is a lot more that could be added as tests, but trying to ration my time.